packages/frontend/src/components/SchedulerModals/SchedulerModalBase/index.tsx

```diff
@@ -36,7 +36,10 @@ const SchedulersModalBase: FC<
                 slackQuery.data?.slackTeamName !== undefined &&
                 !slackQuery.isError &&
                 hasRequiredScopes(slackQuery.data);
-            return isValidSlack ? States.SUCCESS : States.ERROR;
+            // TODO: remove forceSuccess
```

Comment:
This is to allow reviewing without enabling slack.
I will remove this after approval.